.circleci/config.yml

```diff
@@ -0,0 +1,35 @@
+version: 2
+jobs:
+  test:
+    docker:
+      - image: circleci/ruby:10 #where is it coming from ?
```

Comment:
https://circleci.com/docs/2.0/circleci-images/#ruby

Reply:
Once rebased with `master` after #13 gets merged in, it's gonna be fine